### PR TITLE
Kubeflow 1.3 release blog

### PR DESCRIPTION
**Pull request by castrojo**
OK 2nd PR after major revisions, should I squash this?

### CODE REVIEW

**Comment by castrojo** [comment]
I am still missing something, the image is not displaying, I've put the image in the right place and have checked the URLs and it should be working (like it did on the survey blog post), anyone have any clues as to why the image isn't working this time? I'm all out of ideas.

**Comment by jbottum** [comment]
@hamelsmu Hey Hamel, would you please see @castrojo comment above about image not displaying, and perhaps provide any ideas on how to fix.   thank you.

**Comment by hamelsmu** [comment]
Which image is not showing up, exactly?

File: _posts/2021-04-13-kubeflow-1.3-release.md
Comment (hamelsmu) [inline _posts/2021-04-13-kubeflow-1.3-release.md:19]:
```suggestion
![survey_results](/images/2021-04-13-kubeflow-1.3/image1.png)
```

Comment (hamelsmu) [reply _posts/2021-04-13-kubeflow-1.3-release.md:19]: ☝🏽 I think you need to be using an absolute link if I'm not mistaken.

**Review by hamelsmu — APPROVED** [review]
LGTM

You were supposed to use an absolute link rather than a relative link.  I fixed this for you.

**Comment by jbottum** [comment]
@hamelsmu thanks for unblocking us.   question - this blog shows up on my view of blog.kubeflow.org.   This post is not ready to be published.   Is it public / published ?  If so, we can hide it until we get all of the changes and approvals finished?  Thanks again for your prompt actions!

**Comment by hamelsmu** [comment]
Doesn’t show up for me

**Comment by castrojo** [comment]
Ok this is all set afaict content-wise, just holding until the actual release ~4/20, will adjust dates according to the actual release as I get reports from Yannis.

**Comment by theadactyl** [comment]
Awesome - let me know when I should push this through. Thanks for driving
this Josh!

**Comment by jbottum** [comment]
@castrojo we need to modify the 2nd line...In addition to new User Interfaces (UIs), which improve ML workflows for pipeline building, model tuning, serving and monitoring...      change will be.... In addition to new User Interfaces (UIs), which improve ML workflows for pipeline building, volume management, model tuning, model measurements and visualizations...

**Comment by jbottum** [comment]
@andreyvelich would you please review the Katib section and please let me know if there is a better Katib Tour video that we can provide (please provide the url to the improved video tour if you have a good suggestion).   thank you

**Comment by jbottum** [comment]
@kimwnasptd Would you please review the volume management section of this blog post and provide ack ?

File: _posts/2021-04-20-kubeflow-1.3-release.md
Comment (andreyvelich) [inline _posts/2021-04-20-kubeflow-1.3-release.md:30]:
```suggestion
    *   The Katib UI is integrated with the central dashboard and streamlines hyperparameter tuning by presenting a visualization graph and a table that compares each trial’s performance along with its hyperparameters. You can also review the details of each trial’s algorithm, metrics collector and yaml. (Project [1](https://github.com/kubeflow/katib/projects/1))
```

File: _posts/2021-04-20-kubeflow-1.3-release.md
Comment (andreyvelich) [inline _posts/2021-04-20-kubeflow-1.3-release.md:84]: Do we need to say that Katib support k8s>=1.18 in Kubeflow 1.3 ?

**Review by andreyvelich — COMMENTED** [review]
Thank you for writing this @castrojo!

**Comment by andreyvelich** [comment]
> @andreyvelich would you please review the Katib section and please let me know if there is a better Katib Tour video that we can provide (please provide the url to the improved video tour if you have a good suggestion). thank you

@jbottum I believe you took the best link to show some new Katib UI features.
We haven't recorded new demo yet and the new UI is still under development. What do you think @kimwnasptd ?

File: _posts/2021-04-20-kubeflow-1.3-release.md
Comment (castrojo) [reply _posts/2021-04-20-kubeflow-1.3-release.md:84]: It wouldn't hurt, I can add it, where exactly should I put it?

Comment (andreyvelich) [reply _posts/2021-04-20-kubeflow-1.3-release.md:84]: @jbottum What do you think ? Is it a valuable feature that we need to mention in this Blog ?

**Comment by kimwnasptd** [comment]
> @kimwnasptd Would you please review the volume management section of this blog post and provide ack ?

@jbottum @castrojo the section looks fine, thanks!

> We haven't recorded new demo yet and the new UI is still under development. What do you think @kimwnasptd ?

@andreyvelich - @castrojo and I created a video that makes a tour of the new UI a couple weeks ago: https://www.youtube.com/watch?v=1DtjB_boWcQ. We left it upstream-branded so that we could reuse them for things like this. However it's two weeks old already. Jorge and I have the process for making these pretty quickly and easily, so if you want to make a new one we could do it or use this as a placeholder for now and replace at a future date. Let me know!

**Comment by jbottum** [comment]
@animeshsingh We plan to issue the 1.3 Blog post on Tuesday and wanted to give you a heads-up.   Please let us know if you have any recommendations.   Thanks!

**Comment by andreyvelich** [comment]
> > @kimwnasptd Would you please review the volume management section of this blog post and provide ack ?
> 
> @jbottum @castrojo the section looks fine, thanks!
> 
> > We haven't recorded new demo yet and the new UI is still under development. What do you think @kimwnasptd ?
> 
> @andreyvelich - @castrojo and I created a video that makes a tour of the new UI a couple weeks ago: https://www.youtube.com/watch?v=1DtjB_boWcQ. We left it upstream-branded so that we could reuse them for things like this. However it's two weeks old already. Jorge and I have the process for making these pretty quickly and easily, so if you want to make a new one we could do it or use this as a placeholder for now and replace at a future date. Let me know!

Thank you for recording this @kimwnasptd! I think we can use your video in this Blog for now.
As you said, once we finish the new UI implementation we can just replace it.

**Comment by castrojo** [comment]
Reviewers, please do not commit this until docs are ready, which should be this issue: https://github.com/kubeflow/website/issues/2546

If the docs aren't ready I will revise this PR to publish at a future date.

**Comment by castrojo** [comment]
@Bobgy as discussed in the release sync, final push with updated getting started link from Rui, over to you.

Thanks  and see you for 1.4!

**Comment by Bobgy** [comment]
/lgtm
Thank you for the great blogpost!

**Pull request merged**
